freqtrade/freqai/data_drawer.py

```diff
@@ -163,37 +164,67 @@ def load_historic_predictions_from_disk(self):
         Locate and load a previously saved historic predictions.
         :return: bool - whether or not the drawer was located
         """
-        exists = self.historic_predictions_path.is_file()
+        exists = self.historic_predictions_folder.exists()
+        convert = self.historic_predictions_path.is_file()
+
         if exists:
             try:
-                with self.historic_predictions_path.open("rb") as fp:
-                    self.historic_predictions = cloudpickle.load(fp)
+                self.load_historic_predictions_from_folder()
                 logger.info(
                     f"Found existing historic predictions at {self.full_path}, but beware "
                     "that statistics may be inaccurate if the bot has been offline for "
                     "an extended period of time."
                 )
             except EOFError:
                 logger.warning(
-                    'Historical prediction file was corrupted. Trying to load backup file.')
-                with self.historic_predictions_bkp_path.open("rb") as fp:
-                    self.historic_predictions = cloudpickle.load(fp)
-                logger.warning('FreqAI successfully loaded the backup historical predictions file.')
+                    'Historical prediction files were corrupted. Trying to load backup files.')
+                self.load_historic_predictions_from_folder()
```

Comment:
Nothing changed between the call in the try block and here - why would we expect loading to work now (it's not changing the path to a "backup dir" or something similar...)


that said - i think it might make sense to move the try/catch block into the loop within `load_historic_predictions_from_folder` instead.
if we assume a corrupted file - we might still be able to load the remaining files?

Reply:
This was a copy paste mistake, good catch.